### DISCUSSION TITLE
[ask] to add chain solana

### DISCUSSION THREAD

**Discussion by ebennaga** [WalletConnect/walletconnect-monorepo#3541] [Developer Support] [answered]
i want to add chains solana, what should i do? please,give me example code.

thx

**Comment by mshahjahan-mufin** [comment, marked as answer]
@ebennaga 
Please mention your stack or technologies that you are using, if you do not understand the example, open the WalletConnect git repo and search for example directory, you will see how Solana Chain can be added

Here is an example:
 const ChainMetadata(
type: ChainType.solana,
chainId: 'solana:4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ',
 name: 'Solana',
logo: 'TODO',
 color: Colors.black,
rpc: [
  "https://api.mainnet-beta.solana.com",
   "https://solana-api.projectserum.com",
],
  ),
